plugins/metrics/epochcommittment.go

```diff
@@ -1,28 +1,168 @@
 package metrics
 
 import (
+	"github.com/iotaledger/goshimmer/packages/protocol/chainmanager"
 	"sync"
+	"sync/atomic"
+	"time"
 
 	"github.com/iotaledger/hive.go/core/generics/event"
+	"github.com/iotaledger/hive.go/core/generics/shrinkingmap"
 
 	"github.com/iotaledger/goshimmer/packages/core/commitment"
+	"github.com/iotaledger/goshimmer/packages/core/epoch"
+	"github.com/iotaledger/goshimmer/packages/protocol/models"
 )
 
 var (
-	lastCommittedEpoch      = new(commitment.Commitment)
+	lastCommitment          = new(commitment.Commitment)
 	lastCommittedEpochMutex sync.RWMutex
+
+	acceptedBlocksCount       atomic.Int32
+	acceptedTransactionsCount atomic.Int32
+	activeValidatorsCount     atomic.Int32
```

Comment:
Why are these `atomic.Int32`? It seems that they are protected by a mutex anyway?